<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tabs</class>
 <widget class="QWidget" name="Tabs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>487</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="m_pPages">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="Pages::Basic" name="dlgBasic" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="Pages::Advanced" name="dlgAdvanced" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="Pages::Network" name="dlgNetwork" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>Devices</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="Pages::Devices" name="dlgDevices" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Codecs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="Pages::Codecs" name="dlgCodec" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Credentials</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="Pages::Credentials" name="dlgCredentials" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Security</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="Pages::Security" name="dlgSecurity" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Ringtones</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="Pages::RingTone" name="dlgRingtone" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Profile</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QWidget" name="dlgProfileHolder" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>Wizard</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Pages::Basic</class>
   <extends>QWidget</extends>
   <header>conf/accountpages/basic.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Pages::Advanced</class>
   <extends>QWidget</extends>
   <header>conf/accountpages/advanced.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Pages::Network</class>
   <extends>QWidget</extends>
   <header>conf/accountpages/network.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Pages::Codecs</class>
   <extends>QWidget</extends>
   <header>conf/accountpages/codecs.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Pages::RingTone</class>
   <extends>QWidget</extends>
   <header>conf/accountpages/ringtone.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Pages::Credentials</class>
   <extends>QWidget</extends>
   <header>conf/accountpages/credentials.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Pages::Security</class>
   <extends>QWidget</extends>
   <header>conf/accountpages/security.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Pages::Devices</class>
   <extends>QWidget</extends>
   <header>conf/accountpages/devices.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
